Doc/library/ctypes.rst

```diff
@@ -2083,6 +2096,19 @@ Utility functions
    .. availability:: Windows
 
 
+.. function:: dllist()
+   :module: ctypes.util
+
+   Try to provide a list of paths of the shared libraries loaded into the current
+   process.  These paths are not normalized or processed in any way.  The function
+   can raise :exc:`OSError` if the underlying platform APIs fail.
+
+   The first element of the list is typically the path of the current executable
+   file.  The exact functionality is system dependent.
```

Comment:
```suggestion
   The exact functionality is system dependent.

   On most platforms, the first element of the list represents the current
   executable file. It may be an empty string.
```